cli/ceph/fs/OWNERS

```diff
@@ -0,0 +1,15 @@
+---
+# Based on the information found
+# https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md
+
+approvers:
+  - neha-gangadhar
+  - AmarnatReddy
+  - Manimaran-MM
+
+reviewers:
+  - sumabai
+  - hkadam134
```

Comment:
The number of reviewers are less than the approvers... this isn't a good practice.